ios/Configuration/Parameters.swift

```diff
@@ -21,6 +21,8 @@ internal enum Events: String, CaseIterable {
     case didCheckBalance = "didCheckBalanceCallback"
     case didRequestOrder = "didRequestOrderCallback"
     case didCancelOrder = "didCancelOrderCallback"
+    case didBinLookup = "didBinLookupCallback"
+    case didBinValue = "didBinValueCallback"
```

Comment:
`didBinValue` feels incomplete. `didChangeBinValue` or `didBinValueChange` perhaps?

Reply:
Good one!

Reply:
Fixed in [47c4a93](https://github.com/Adyen/adyen-react-native/pull/592/commits/47c4a9318c09f8d92c1f2ccfd5f2e3b00750f4bd)